README.md

```diff
@@ -65,6 +65,7 @@ parameters:
     bin_dir: "./vendor/bin"
     git_dir: "."
     stop_on_failure: false
+    ignore_unstaged_changes: false
```

Comment:
I am also thinking about setting the default value to true. 
The only downside is that changes are not available during the commit (when tests run for a long time e.g.) Maybe I should add some additional output like 'Detected unstaged changes - temporarily stash them'. This way it's clear for the user what happens.